library/BIGBITMAPS

```diff
@@ -1,15 +1,13 @@
 (DEFINE-FILE-INFO PACKAGE "INTERLISP" READTABLE "XCL" BASE 10)
 
-(FILECREATED " 9-Jul-2022 09:41:26" 
-|{DSK}<Users>kaplan>Local>medley3.5>working-medley>library>BIGBITMAPS.;12| 108851 
+(FILECREATED "31-Jul-2023 13:39:50" |{WMEDLEY}<library>BIGBITMAPS.;13| 109376 
+
+      :EDIT-BY |rmk|
 
       :CHANGES-TO (VARS BIGBITMAPSCOMS)
-                  (FNS \\GENERIC.DSPCREATE.DESTINATION.BITMAP?.BIGBM \\GENERIC.DSPCREATE.BIGBM 
-                       \\DSPCREATE.BIGBM)
-                  (MACROS |\\SFInvert|)
+                  (FNS BIGBITMAPEQUAL)
 
-      :PREVIOUS-DATE "26-Oct-2021 14:51:38" 
-|{DSK}<users>kaplan>local>medley3.5>working-medley>library>BIGBITMAPS.;6|)
+      :PREVIOUS-DATE " 9-Jul-2022 09:41:26" |{WMEDLEY}<library>BIGBITMAPS.;12|)
 
 
 ; Copyright (c) 1991, 1993-1994 by Venue.
```

Comment:
please load SYSEDIT when editing system files. It among other things resets the COPYRIGHTFLG.